app/src/main/java/blockeq/com/stellarwallet/WalletApplication.kt

```diff
@@ -32,11 +34,28 @@ class WalletApplication : MultiDexApplication() {
 
         var session : Session? = null
 
+        var currAssetCode: String = Constants.LUMENS_ASSET_TYPE
+        var currAssetName: String = Constants.LUMENS_ASSET_NAME
+        var currAssetIssuer: String = ""
+
         var appReturnedFromBackground = false
 
         fun applicationContext(): WalletApplication {
             return instance!!
         }
+
+        fun getAssetCode() : String {
```

Comment:
@Daniel-Wang Apologies - i should have been more specific. I feel creating another class such as `UserSession` and putting related functions in there would be more suitable. We can also rename `getAssetCode` to `getCurrentAssetCode`. Similarly, `getAvailableBalance` should also be in `UserSession`.

Reply:
Okay gotcha!
So that would also have the 3 curr asset vars right?

Reply:
@Daniel-Wang yes!

Reply:
@MeghaBambra how does this look now?